package.json

```diff
@@ -21,6 +21,9 @@
     "babel:browser": "cross-env BABEL_ENV=browser babel --out-dir dist/browser src",
     "babel:node": "cross-env BABEL_ENV=node babel --out-dir dist src",
     "babel:node:async": "cross-env BABEL_ENV=node:async babel --out-dir dist src/node",
+    "babel:watch": "npm-run-all -p babel:watch:*",
+    "babel:watch:browser": "npm run babel:browser -- --verbose --watch",
+    "babel:watch:node": "npm run babel:node -- --verbose --watch",
```

Comment:
I think `--` doesn't work in Windows. You could use cross-env to make sure it works

Reply:
It's the same as we have in `package.json` for Embark proper. Does that one work?

Reply:
Oh, yes it works. Ok then, I must be mistaken with something else.